papis/database/__init__.py

```diff
@@ -1,6 +1,6 @@
 from typing import Optional, Dict
 
-from .base import Database
+from .base import Database as Database  # Explicit reexport to maky mypy happy
```

Comment:
Can you remove this and change the places in `test_papis.py` to use `papis.database.cache.Database` instead? 

This should probably be under a `TYPE_CHECKING` guard or something like that since it's not meant to be exported.

Reply:
OK, done. Not sure what you mean by a `TYPE_CHECKING` guard?

Reply:
Ah, I just meant that this import is only used for type annotations, so it could be imported under a [TYPE_CHECKING guard](https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING) like here:
https://github.com/papis/papis/blob/241eef6371f3a37bbaef0eb327873e8b765f5168/papis/arxiv.py#L37-L38

It's fine to leave it as is though! Just thinking out loud :grin:

Reply:
@hseg This doesn't seem to be updated yet. Did you forget to push it?